### ISSUE TITLE
Color in editor not appearing after load

### ISSUE THREAD

**Issue by javizqh** [JdeRobot/bt-studio#128]
Accent color on editor not appearing randomly after load, and then functioning correctly after something is updated

**Comment by OscarMrZ** [comment]
Do we really need random accent color editor? I think we should focus on setting a stable palette

**Comment by javizqh** [comment]
It only happens the first time after launching the frontend

**Comment by OscarMrZ** [comment]
Probably this is due to react being unable to track the value of actionNodesData and update the component, as the setter hook is not being used. This produces the random behavior, as per the react docs "you should treat any JavaScript object that you put into state as read-only". And we aren't doing that:

```js
actionNodesData[name]["output"] = actionNodesData[name]["output"].concat([key]);
```

Related: 
* https://react.dev/learn/updating-arrays-in-state. 
* https://react.dev/learn/updating-objects-in-state

We should revisit this in the future, doing it in the standard way. For the moment I have implemented a hotfix in #168
